src/configuration/js-team/index.js

```diff
@@ -0,0 +1,35 @@
+import Linux64 from './linux64';
+import MacConfig from './macOS';
+import Win7Config from './win7';
+import Win10Config from './win10';
+import AndroidConfig from './android';
+
+const platforms = {
+  linux64: {
+    label: 'Linux 64bit',
+    platform: 'linux64',
```

Comment:
I realize that the name of this property is probably at fault for making this more complicated to understand.
Let's name it `pickerKey` or `dropdownKey` or `viewKey`. The one that will help you most.